mix.exs

```diff
@@ -36,8 +36,10 @@ defmodule ExMachina.Mixfile do
     [
       {:ex_doc, "~> 0.14", only: :dev},
       {:earmark, ">= 0.0.0", only: :dev},
-      {:ecto, "~> 2.1", optional: true},
-      {:postgrex, ">= 0.0.0", only: [:test]},
+      {:ecto, "~> 2.2 or ~> 3.0", optional: true},
+      {:jason, "~> 1.0", optional: true},
```

Comment:
Do we need to include `jason` as part of upgrading to 3.0? If so, is it safe to remove `poison`?

Reply:
Looking at code, Poison is only used it docs so I'd suggest to remove it from mix.exs and optionally promote Jason as it's recommended by Ecto & Phoenix.